dist/templates/k8s.ovn.org_egressfirewalls.yaml.j2

```diff
@@ -0,0 +1,122 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: (unknown)
```

Comment:
why is this printing the kubebuilder version as `unknown`?

Reply:
I think he must have built it with a custom build of `controller-gen`, could you re-build with the officially downloaded version of `controller-gen` by `hack/update-codegen.sh` and update this @JacobTanenbaum ? Or remove this altogether?

Reply:
I fixed the template using controller-gen it should be all good now